docs/conf.py

```diff
@@ -20,7 +20,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'PyOData'
-copyright = '2019 SAP SE or an SAP affiliate company'
+copyright = '2021 SAP SE or an SAP affiliate company'
```

Comment:
This could be nice, separate PR, immediately merged.

Reply:
Nah, you update copyright, when you touch the code. This is change is OK.

Reply:
Moved to separate commit to make it easier to cherry pick this change.